app/scripts/controllers/reviewRegistration.js

```diff
@@ -249,8 +249,13 @@ angular.module('confRegistrationWebApp')
     $scope.mergeAndConfirmRegistration = function () {
       // Pay for the spouse's registration before merging it with the spouse
       $scope.submittingRegistration = true;
-      registration.submitPayment($scope.currentPayment, $scope.spouseRegistration, $scope.acceptedPaymentMethods()).then(function () {
-        return registration.mergeWithSpouse(currentRegistration, $scope.spouseRegistration);
+
```

Comment:
@dougworner Payment was invalid due to overpayment. This happened since the 'husband' registration was already paid in full when we tried to submit a payment for the 'wife'. If we first merge the wife registrant onto the husband registration, then the amount due increases and payment can be accepted.